### ISSUE TITLE
Can we move the entrypoint script into the image?

### ISSUE THREAD

**Issue by trombonekenny** [eddelbuettel/pl-r-demos#4]
I notice these questions source the entrypoint script from inside the course space

`        "entrypoint": "/grade/serverFilesCourse/r_autograder/run.sh",`

Is that static enough that we could safely move a copy into the autograder image? One less thing for adopters to need to have locally, and seems to be what the other graders do.

**Comment by trombonekenny** [comment]
Ah, I see it's already there at `/r_autograder/run.sh`?

https://github.com/PrairieLearn/PrairieLearn/blob/a64329cbe01df77d551bf415622757d5bfb5fd27/graders/r/Dockerfile#L86

**Comment by eddelbuettel** [comment]
Yes, and it is a 'six of one, half a dozen of another' problem as it is good to have the autograder shipped.  But if you noticed last minute that you fgorgot to include one package update in the container it is also good to call an external one.  We should document both.

And some of these questions are ... old.  I'll try to have a pass over them too.

**Comment by trombonekenny** [comment]
Confirmed. We just need to update the questions to point the script inside the image. I think the /serverFilesCourse/datasets folder is a good example so we can leave it, although I think Nathan would prefer that large static datasets are included in the image and not passed around through serverFilesCourse

(Not calling penguins.csv large, just not wanting to set a precedent where someone puts >100MB datasets in serverFilesCourse or with the question)

**Comment by eddelbuettel** [comment]
Note that I talked about code, not data.  I sometimes need to update the whole autograder, and "in PrairieLearn proper" is great ("it is everwhere") but I am then at the whim of a PR merge and rebuild.  As I said, it is of size six both cases.  Both use cases exists.
